plugins/@grouparoo/app-templates/src/destination/batch/exportProfiles.ts

```diff
@@ -450,10 +456,21 @@ async function lookupDestinationIds(
       continue;
     }
 
-    const { foreignKeyValue, oldForeignKeyValue } = exportedProfile;
+    let foreignKeyValue: string;
+    switch (fkType) {
+      case "new":
+        foreignKeyValue = exportedProfile.foreignKeyValue;
+        break;
+      case "old":
+        foreignKeyValue = exportedProfile.oldForeignKeyValue;
```

Comment:
If this case if they are the same, you don't have to do it.
then note that the total could be zero.

Reply:
oldForeignKeyValue is only set if they are different. See `assignForeignKeys()`:
https://github.com/grouparoo/grouparoo/blob/ce6dc7b70ccd6a77fa654f373718b29cf394b730/plugins/%40grouparoo/app-templates/src/destination/batch/exportProfiles.ts#L571-L577

Reply:
ah ha. perfect.